docs/content/doc/help.fr-fr.md

```diff
@@ -0,0 +1,13 @@
+---
+date: "2017-01-20T15:00:00+08:00"
+title: "l'aider"
+slug: "help"
+weight: 5
+toc: false
+draft: false
+menu:
+  sidebar:
+    name: "l'aider"
```

Comment:
Oh can we please just move these to crowd-in already...

I know I argued against it previously - somewhat hoping we'd process the English translation to split it automatically but I was wrong. It's gotta go.

Reply:
I'm not against moving them to crowdin.  
I only added these pages because they are tiny and would still create a fully functional search in their own locale.

Reply:
Yeah don't worry I wasn't wanting you to change this now - was simply a wail into the ether